### PR TITLE
[tests-only][full-ci] Exclude test suites that are not meant to run on oCIS

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Added file `suites-not-to-run-on-ocis.txt` which includes a list of test suites that are not to run in `oCIS`.
While running WebUI tests in parts (in oCIS), these suites will be excluded.

## Related Issue
- fixes https://github.com/owncloud/ocis/issues/2356

## How Has This Been Tested?
- test environment: CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by saw-jan** [comment]
Tested:
https://drone.owncloud.com/owncloud/ocis/6554/6/6
https://drone.owncloud.com/owncloud/ocis/6527

File: tests/acceptance/run.sh [hunk 1]
```diff
@@ -82,7 +82,16 @@ then
 	done
 elif [[ -n  "${RUN_PART}" ]]
 then
-	ALL_SUITES=$(find "${FEATURES_DIR}"/ -type d | sort | rev | cut -d"/" -f1 | rev)
+  	ALL_SUITES=$(find "${FEATURES_DIR}"/ -type d | sort | rev | cut -d"/" -f1 | rev | grep -v '^[[:space:]]*$')
```
Comment (phil-davis) [inline tests/acceptance/run.sh:85]: @saw-jan please write some explanation here in GitHub about why you needed to add the `grep -v '^[[:space:]]*$'` to the end of this command pipeline.

Also, the indent has changed - but that is not a big problem.

**Review by phil-davis — APPROVED** [review]

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18699/24/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:114

File: tests/acceptance/run.sh (hunk 1 above)
Comment (saw-jan) [reply tests/acceptance/run.sh:85]: `grep -v '^[[:space:]]*$'` removes empty new lines

**Comment by phil-davis** [comment]
@saw-jan please do a commit id bump in `owncloud/ocis` and we can check the CI to see that it does run the suites that we expect.

**Pull request merged**